tests/components/conversation/test_default_agent.py

```diff
@@ -441,6 +441,17 @@ async def test_nevermind_item(hass: HomeAssistant) -> None:
     assert not result.response.speech
 
 
+@pytest.mark.usefixtures("init_components")
+async def test_respond_intent(hass: HomeAssistant) -> None:
+    """Test HassRespond intent through the default agent."""
+    result = await conversation.async_converse(hass, "hello", None, Context())
```

Comment:
If possible, please also add a test in `tests/components/intents/test_init.py` calling `helpers.intent.async_handle`.

Reply:
Added a test for the new intent as well as "nevermind" :+1: